src/lib/appwrite.js

```diff
@@ -0,0 +1,10 @@
+import { Client, Account, Databases, Health } from "node-appwrite";
```

Comment:
Do we want to use the web SDK or the Node SDK

Reply:
We'll use the Web SDK for now I think, maybe later we could do SSR